api/v1alpha1/volumesnapshotrestore_types.go

```diff
@@ -74,6 +88,7 @@ const (
 	SnapMoverRestorePhaseCleanup VolumeSnapshotRestorePhase = "Cleanup"
 )
 
+// Could we use the same type from backup?
```

Comment:
the enum values are the same, but the key have different names. There is any advantages of having 2 similar enums?